.github/workflows/release.yml

```diff
@@ -16,6 +16,7 @@ env:
   # renovate: datasource=github-releases depName=kubernetes-sigs/controller-tools
   CONTROLLER_TOOLS_VERSION: "v0.12.1"
   SCHEDULER_COMPATIBLE_K8S_VERSION: "v0.24.3"
+  PUBLISHABLE_ITEMS: '[".", "lifecycle-operator", "metrics-operator", "klt-cert-manager", "runtimes/deno-runtime", "runtimes/python-runtime"]'
```

Comment:
why the `.` (dot)? 
Furthermore, I think we should remove `klt` from `klt-cert-manager`

Reply:
the `.` is a catch-all for everything that is not monorepo-ized i.e. examples, docs, helm chart

Reply:
maybe worth having a comment in the switch case a few lines below to describe what artifacts are collected under `.` so it is cristal clear :)